examples/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ p8e {
             location.signingPrivateKey = System.getenv("SIGNING_PRIVATE_KEY")
             location.chainId = System.getenv("CHAIN_ID")
             location.txBatchSize = "5"
+            location.txFeeAdjustment = "2.0"
```

Comment:
I'm not really sure why I was seeing fee problems until I moved this up. 🤷

Reply:
:same: